### PR TITLE
Added cmake switch to disable usage of OpenMP

### PR DESCRIPTION
**Pull request by HannesSommer**

### CODE REVIEW

**Comment by pomerlef** [comment]
Looks good to me

File: CMakeLists.txt [hunk 1]
```diff
@@ -49,14 +49,17 @@ if(MSVC AND (MSVC_VERSION LESS 1600))
...
-if (OPENMP_FOUND)
-  add_definitions(-fopenmp -DHAVE_OPENMP)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-  if (CMAKE_COMPILER_IS_GNUCC)
-    set(EXTRA_LIBS ${EXTRA_LIBS} gomp)
-  endif(CMAKE_COMPILER_IS_GNUCC)
-endif(OPENMP_FOUND)
+set(USE_OPEN_MP "true" CACHE BOOL "Set to OFF to not use OpenMP")
+if(USE_OPEN_MP)
+	find_package(OpenMP)
```
Comment (simonlynen) [inline CMakeLists.txt]: please make the indentation consistent with the rest of the file. (2 space indent)

**Comment by simonlynen** [comment]
apart from indentation, lgtm.

**Comment by HannesSommer** [comment]
Thanks for the feedback. I've made the block I touched as consistent with the rest as I could think of. 

The sad truth is that regarding white-spaces this file is already in a nightmare like state :). 

So the question is what is the underlying rule for this project?
A language ignorant statistics on the files there is a clear majority in favour for tab indentation. Which also fits the personal preference of the main author, @stephanemagnenat , IMO. So I guess it was originally planned to be tab intended. This also fits with the output of git blame for this very file the older parts are tab indented.

I'd suggest to merge this as is and have a follow up issue clarifying things and fixing existing indentation.
I've opened issue #54.

**Pull request merged**